x-pack/solutions/security/packages/ecs_data_quality_dashboard/impl/data_quality_panel/actions/chat/index.tsx

```diff
@@ -19,19 +20,27 @@ import {
 import { useDataQualityContext } from '../../data_quality_context';
 import { ASK_ASSISTANT } from './translations';
 
-const StyledLinkText = styled.span`
-  display: flex;
-  gap: ${({ theme }) => theme.eui.euiSizeXS};
-`;
+const useStyles = () => {
+  const { euiTheme } = useEuiTheme();
+
+  return {
+    linkText: css({
+      display: 'flex',
+      gap: euiTheme.size.xs,
+    }),
```

Comment:
using object notation is recommended for 2 reasons:

1. emotion docs advise to use it for typescript https://emotion.sh/docs/typescript#emotionreact

![image](https://github.com/user-attachments/assets/002354f2-3bfc-4850-b9c6-886cdf0ce40b)

2. Free Autocomplete of keys and values.